tests/integration/linodes/helpers_linodes.py

```diff
@@ -205,10 +237,30 @@ def create_linode_and_wait(
         )
     linode_id = output
 
-    # wait until linode is running
-    assert (
-        wait_until(linode_id=linode_id, timeout=240, status="running"),
-        "linode failed to change status to running",
-    )
+    # wait until linode is running, wait_until returns True when it is in running state
+    result = (wait_until(linode_id=linode_id, timeout=240, status="running"),)
+
+    assert result, "linode failed to change status to running"
 
     return linode_id
+
+
+def set_backups_enabled_in_account_settings(toggle: "bool"):
```

Comment:
```suggestion
def set_backups_enabled_in_account_settings(toggle: bool):
```
We can use `bool` type here.